com.unity.multiplayer.mlapi/Runtime/Serialization/BitWriter.cs

```diff
@@ -52,11 +52,14 @@ internal Stream GetStream()
         /// <param name="value">The object to write</param>
         public void WriteObjectPacked(object value)
         {
-            if (value == null || value.GetType().IsNullable())
+            // Check unitys custom null checks
+            bool isNull = value == null || (value is UnityEngine.Object && ((UnityEngine.Object)value) == null);
```

Comment:
Since this gets called a lot, I wonder how expensive the type checking and casting is

Reply:
Also, the IsNullable call is only used in one other place, "ReadObjectPacked".  Should similar logic be over there too?  Can we eliminate IsNullable?

Reply:
Oh, ok, I see now IsNullable *is* getting checked in BitReader.cs.